README

```diff
@@ -132,12 +132,17 @@ How to build unreleased version
 -------------------------------
 
   $ git clone --recursive https://github.com/uim/sigscheme.git
```

Comment:
`--recursive` does `git submodule init` and `git submodule update` automatically.